newtmgr/config/ble_config.go

```diff
@@ -40,13 +40,17 @@ type BleConfig struct {
 	OwnAddrType bledefs.BleAddrType
 	OwnAddr     bledefs.BleAddr
 
+	// Connection timeout, in seconds.
+	ConnTimeout float64
+
 	BlehostdPath   string
 	ControllerPath string
 }
 
 func NewBleConfig() *BleConfig {
 	return &BleConfig{
 		OwnAddrType:  bledefs.BLE_ADDR_TYPE_RANDOM,
+		ConnTimeout:  10.0,
```

Comment:
Would it make sense to have this timeout be the same as the newtmgr timeout, unless explicitly set otherwise? I guess these have been disjoint even until now, so probably not worth adjusting that logic within this PR :)